docs/src/api/params.md

```diff
@@ -1716,7 +1716,7 @@ Consider the following config:
 import { defineConfig } from '@playwright/test';
 
 export default defineConfig({
-  snapshotPathTemplate: '__screenshots__{/projectName}/{testFilePath}/{arg}{ext}',
+  snapshotPathTemplate: '__screenshots__/{projectName}/{testFilePath}/{arg}{ext}',
```

Comment:
That is not a typo, it means it is optional and is only used when there is a non-empty project name.

Reply:
You're right. I didn't know that. I thought it's just a typo. Thank you for your comments!